packages/components/src/popover/index.js

```diff
@@ -188,6 +189,30 @@ const Popover = (
 		resize = ! __unstableForcePosition;
 	}
 
+	if ( anchorRef !== undefined ) {
```

Comment:
Do we need to be so strict about `undefined`? Wouldn't a simple truthy check be enough - `if ( anchorRef )`?

Reply:
We want to make sure that the `anchorRef` prop is not used, i.e. that no value is passed to it at all. Using this `undefined` check makes sure that even _truthy_ values will trigger the deprecation warning